tracee-rules/stats/stats.go

```diff
@@ -0,0 +1,7 @@
+package stats
+
+type RulesStats struct {
```

Comment:
These are metrics exposed by Tracee-rules right? Why don't we just call this struct Metrics? Also Rules prefix is somehow redundant because this code lives in the tracee-rules package

Reply:
Resolved